src/app/component/user/components/profile/profile-service/profile.service.ts

```diff
@@ -28,6 +29,11 @@ export class ProfileService {
     return this.http.get<CardModel>(`${this.backEnd}facts/dayFact/2`);
   }
 
+  public getShoppingList(): Observable<ShoppingList[]> {
+    this.setUserId();
+    return this.http.get<ShoppingList[]>(`${this.backEnd}goals/shoppingList/${this.userId}/language/en`);
```

Comment:
General: it is better to leave an empty line before the 'return' for more readability

Reply:
done